### PR TITLE
New HC to check the quality of alignments.

### PR DESCRIPTION
**Pull request by mateuspatricio**
This HC checks alignments produced by EG.

### CODE REVIEW

**Comment by mateuspatricio** [comment]
Since the branch will happen tomorrow. I' adding this HC in here to be merged.

**Comment by muffato** [comment]
This PR should be against `master`, not `release/95`

**Comment by muffato** [comment]
Why do we need this new group ? There is already `ComparaGenomic` for the alignments

**Comment by mateuspatricio** [comment]
@muffato, there are a few things in ComparaGenomic that does not apply for the EG alignments.

**Comment by muffato** [comment]
I thought that an `if` structure like in https://github.com/Ensembl/ensj-healthcheck/blob/release/95/src/org/ensembl/healthcheck/testcase/compara/CheckConservationScore.java#L62-L80 would tailor the test to each database. I'll run the main `ComparaGenomic` group on plants and list the tests that have to be changed. Adding yet another group is my least favourite option as I'm sure it would go out of sync with the main one

**Comment by muffato** [comment]
FYI, I've run the `ComparaGenomic` group against the plants database. 2 HCs have failed because I haven't configured the server properly but they would otherwise succeed.
2 HCs have failed because of real errors in the database:
 - the HC I've added on master last week https://github.com/Ensembl/ensj-healthcheck/pull/152 (and thus wasn't used to test the e95 db) which was meant to capture an error in the plants db, so that one is expected and will be fixed in e96
 - an HC I've added on master last month https://github.com/Ensembl/ensj-healthcheck/pull/141 (and thus wasn't used to test the e95 db) which was also meant to capture an errors we had found in the plants db. That one will also be fixed in e96

In summary, all the errors found have to be fixed, all the checks in `ComparaGenomic` apply to Plants, and there is no need to have another group
